src/alert/types.ts

```diff
@@ -14,4 +14,9 @@ export interface AlertProps extends React.HTMLAttributes<HTMLDivElement> {
     showIcon?: boolean | undefined;
     actionLinkIcon?: JSX.Element | undefined;
     sizeType?: AlertSizeType | undefined;
+    /** Custom alert icon to be rendered, replacing the default alert icons */
+    customAlertIcon?: JSX.Element | undefined;
```

Comment:
```suggestion
    customIcon?: JSX.Element | undefined;
```
Since we already know this is an `Alert` component